guides/common/modules/ref_required-red-hat-repositories.adoc

```diff
@@ -0,0 +1,52 @@
+[id="required-red-hat-repositories"]
+= Required Red Hat repositories
+
+You need the following repositories to manage hosts with {RHEL}.
+
+For {RHEL} 10 hosts::
+You require both:
+
+* *{RHEL} 10 for x86_64 - BaseOS (RPMs)*
+* *{RHEL} 10 for x86_64 - AppStream (RPMs)*
+
+For {RHEL} 9 hosts::
+You require both:
+
+* *{RHEL} 9 for x86_64 - BaseOS (RPMs)*
+* *{RHEL} 9 for x86_64 - AppStream (RPMs)*
+
+For {RHEL} 8 hosts::
+You require both:
+
+* *{RHEL} 8 for x86_64 - BaseOS (RPMs)*
+* *{RHEL} 8 for x86_64 - AppStream (RPMs)*
+
+For {RHEL} 7 hosts::
+You require the *{RHEL} 7 Server (RPMs)* repository.
++
+The difference between associating {RHEL} operating system release version with either *7Server* repositories or *7._y_* repositories is that *7Server* repositories contain all the latest updates while *{RHEL} 7._y_* repositories stop getting updates after the next minor version release.
+
+.Additional repositories for provisioning
+To provision hosts by using the Anaconda installer with Kickstart, you additionally need the following repositories:
+
+For {RHEL} 10 hosts::
+You require both:
+
+* *{RHEL} 10 for x86_64 - BaseOS (Kickstart)*
+* *{RHEL} 10 for x86_64 - AppStream (Kickstart)*
+
+For {RHEL} 9 hosts::
+You require both:
+
+* *{RHEL} 9 for x86_64 - BaseOS (Kickstart)*
+* *{RHEL} 9 for x86_64 - AppStream (Kickstart)*
+
+For {RHEL} 8 hosts::
+You require both:
+
+* *{RHEL} 8 for x86_64 - BaseOS (Kickstart)*
+* *{RHEL} 8 for x86_64 - AppStream (Kickstart)*
+
+For {RHEL} 7 hosts::
+You require the *{RHEL} 7 Server (Kickstart)* repository.
+Note that Kickstart repositories only have minor versions.
```

Comment:
Nit: This sounds weird to me, it took me a while to understand that it means you cannot use the `latest` version.

Reply:
I'll probably drop this. RH is not releasing new RHEL 7 versions anymore.